apps/src/codebridge/Console/index.tsx

```diff
@@ -11,6 +11,19 @@ import moduleStyles from './console.module.scss';
 const Console: React.FunctionComponent = () => {
   const codeOutput = useAppSelector(state => state.codebridgeConsole.output);
   const dispatch = useDispatch();
+  const levelId = useAppSelector(state => state.lab.levelProperties?.id);
+  const previousLevelId = useRef(levelId);
+  const appName = useAppSelector(state => state.lab.levelProperties?.appName);
+
+  const systemMessagePrefix = appName === 'pythonlab' ? '[PYTHON LAB] ' : '';
```

Comment:
Should we add a TODO to reference updating this with other appNames eventually?

Reply:
good call, will update